### PR TITLE
cb179 Board mobile version

### PR DESCRIPTION
**Pull request by Brandonz07**
Adjust the dashboard in the mobile version

image

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]

File: src/pages/board/outlets/boardlayout/interface.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { useEffect, useRef, useState } from "react";
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/interface.tsx:1]: The select check is not working. Merge https://inube.atlassian.net/browse/CB-172 into main, merge main into cb179, and verify that everything works.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/board/outlets/boardlayout/interface.tsx [hunk 2]
```diff
@@ -86,67 +152,72 @@ function BoardLayoutUI(props: BoardLayoutProps) {
...
+            <Stack gap={inube.spacing.s200}>
               {!isMobile && (
-                <Text type="label">Ver unicamente los anclados</Text>
+                <Stack gap={inube.spacing.s100} alignItems="center">
+                  <Icon
+                    icon={<MdOutlinePushPin />}
+                    appearance="dark"
+                    size="24px"
+                  />
+                  <Text type="label">Ver unicamente los anclados</Text>
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/interface.tsx:163]: Relocate this into a config file

File: src/pages/board/outlets/boardlayout/interface.tsx [hunk 3]
```diff
@@ -49,31 +56,90 @@ function BoardLayoutUI(props: BoardLayoutProps) {
...
+        direction="column"
+        width={isMobile ? "-webkit-fill-available" : "min(100%,1500px)"}
+      >
+        <StyledInputsContainer $isMobile={isMobile}>
+          <Stack
+            justifyContent="space-between"
+            width={isMobile ? "100%" : "auto"}
+            margin={isMobile ? "s100 s0" : "auto"}
+          >
+            <StyledSearch
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/interface.tsx:95]: After writing a string to search, the search text looks invalid contracting itself, i would block the isExpanded state based on the content or hide the content
image

File: src/pages/board/outlets/boardlayout/interface.tsx [hunk 4]
```diff
@@ -86,67 +152,72 @@ function BoardLayoutUI(props: BoardLayoutProps) {
...
-              orientation={boardOrientation}
-              sectionInformation={filteredRequests}
-              pinnedRequests={pinnedRequests}
-              handlePinRequest={handlePinRequest}
-            />
-          );
-        })}
-      </StyledBoardContainer>
-    </Stack>
+            return (
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/interface.tsx:207]: This return is required? avoid a return into a return, avoid multiple returns for 1 react component. I would try to delete the return in 207 and verify its operation

File: src/pages/board/outlets/boardlayout/interface.tsx [hunk 5]
```diff
@@ -86,67 +152,72 @@ function BoardLayoutUI(props: BoardLayoutProps) {
...
+              <BoardSection
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/interface.tsx:208]: Adjust the components accordingly to the wireframes, this task includes the adjustment of the board section considering that it is not centered in the mobile version.

image

**Review by carin91 — CHANGES_REQUESTED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**